### PR TITLE
Feature/1 calendar

### LINKED ISSUE #1: Create component for calendar
# What we want :telescope:
Calendar for the application.

# Acceptance Criteria :sparkles: 
Able to see the date and events.

# T-Shirt :tshirt:

### PR DESCRIPTION
**Pull request by Bankow**
Create base component for calendar.
Resolves #1 .

### CODE REVIEW

File: src/app/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { Component, OnInit} from '@angular/core';
+
+import { Component, OnInit, OnDestroy } from '@angular/core';
```
Comment (Schieck) [inline src/app/dashboard/dashboard.component.ts]: Please undo this change.

File: src/main.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import 'hammerjs';
+import 'hammerjs';
```
Comment (Schieck) [inline src/main.ts]: Please remove this line.

File: src/app/dashboard/dashboard.component.ts (hunk 1 above)
Comment (Bankow) [reply src/app/dashboard/dashboard.component.ts]: Resolved

File: src/main.ts (hunk 2 above)
Comment (Bankow) [reply src/main.ts]: this duplicated import removed
